tests/e2e/framework/kubernetes.go

```diff
@@ -625,11 +625,6 @@ func (k *KubeInfo) Teardown() error {
 	log.Infof("Deleting namespace %v", k.Namespace)
 	for attempts := 1; attempts <= maxAttempts; attempts++ {
 		if *useOperator {
-			if _, err := util.Shell("kubectl delete ns istio-operator --kubeconfig=%s",
-				k.KubeConfig); err != nil {
```

Comment:
If I understand this correctly, this namespace should already be deleted by line 575.
So, instead of just setting `namespaceDeleted = true`, why not call something like `namespaceDeleted, _ = util.NamespaceDeleted("istio-operator", k.KubeConfig)`?

Reply:
Thanks! Fixed